custom/icds_reports/migrations/0100_add_supervisor_id.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-02-22 07:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0099_service_delivery_report'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='aggregateawcinfrastructureforms',
+            name='supervisor_id',
+            field=models.CharField(max_length=40, null=True),
```

Comment:
these must all be `Text` to match the UCRs so that we get co-location with citusdb